lib/model.js

```diff
@@ -959,6 +959,10 @@ Model.prototype.sync = function(options) {
     , attributes = this.tableAttributes;
 
   return Promise.try(function () {
+    if (options.hooks) {
```

Comment:
@mickhansen / @janmeier Hmm, I am not sure what do you mean with "options.hooks". I followed pattern in other hooks like here https://github.com/sequelize/sequelize/blob/master/lib/model.js#L2053 . It seems that when `sync` method is called (either on model or sequelize instance) then by by default `hooks` field is set on options object and it looks like this:

```js
hooks: { beforeSync: [ [Function] ], afterSync: [ [Function] ] },
```